src/components/cards/SightingsCard.jsx

```diff
@@ -55,36 +55,36 @@ export default function SightingsCard({
     {
       reference: 'date',
       name: 'time',
-      label: 'Date reported',
+      labelId: 'DATE_OF_SIGHTING',
       options: {
         cellRenderer: cellRendererTypes.specifiedTime,
       },
     },
     {
       reference: 'created',
       name: 'created',
-      label: 'Date created',
+      labelId: 'DATE_REPORTED',
```

Comment:
This and DATE_OF_SIGHTING look like their order was swapped. Just confiming that this was intentional.